llm_on_ray/finetune/finetune_config.py

```diff
@@ -71,6 +72,8 @@ class Dataset(BaseModel):
     group: bool = True
     block_size: int = 512
     shuffle: bool = False
+    pad_max: bool = False
+    max_prompt_length: int = 512
```

Comment:
No, GaudiDPOTrainer has max_length parameter, But I will set default max_prompt_length was None. 
```
        max_length: Optional[int] = None,
        max_prompt_length: Optional[int] = None,
        max_target_length: Optional[int] = None,
```
Because GaudiDPOTrainer has default was 128
```
            if max_prompt_length is None:
                warnings.warn(
                    "When using DPODataCollatorWithPadding, you should set `max_prompt_length` in the DPOTrainer's init"
                    " it will be set to `128` by default, but you should do it yourself in the future.",
                    UserWarning,
                )
                max_prompt_length = 128
```